codegen.yml

```diff
@@ -0,0 +1,9 @@
+overwrite: true
+schema: "https://master.staging.saleor.cloud/graphql/"
```

Comment:
use env var for that

Reply:
Also, pointing to the staging api will result in random type changes on builds after core merges